### ISSUE TITLE
Last resort to register

### ISSUE THREAD

**Issue by InqOnThat** [caddyserver/website#356]
**Tried to register on your forum** - I've gone through the registration, and received the Activation Email.  However the "Click here to activate your account" does nothing.  I've tried it on a Windows PC and an Android Phone same result.  Did a "Resend Activation Email" and again same result.  Can you help me with this?

The only contact info you have visible on either your forum or your caddyserver.com website was contact@caddyserver.com.  That gave an me the Catch 22 response -

>  "_The email account you have tried to reach is only used for administrative purposes and is not actively monitored. If you require a response, please redirect your inquiry to the proper category in the Caddy forums: https://caddy.community/_".

If I could respond on the forum, I wouldn't have needed to contact you!

**Comment by mholt** [comment]
Hm, that's odd. What's "does nothing" mean exactly? The link doesn't even open a browser window?

What is your account username? I'll look into it.

**Comment by InqOnThat** [comment]
Thank you for your response.  "nothing" means... it flickers like any web
button does when clicked, but nothing else happens... no window, no
confirmation email, no noise...

I filled out the form with username Inq, email:  ***@***.*** (this
email) and I'm Dennis Cox.

**Comment by mholt** [comment]
Strange, almost as if the email client isn't responding to the touch/click properly. If it were a problem with the forum I'd expect the page to at least open first.

I activated your account for you and bumped your trust level from 0 to 1, since I presume you're not a bot.

Sorry for the trouble! Hope that works now.

**Comment by InqOnThat** [comment]
*"I activated your account for you and bumped your trust level from 0 to 1,
since I presume you're not a bot.*"

😁  Thank you for your lightning response!

*"Strange, almost as if the email client isn't responding to the
touch/click properly."*

Just for your information... and clarity, it wasn't the email page.  This
page button worked fine:
[image: image.png]
... and popped up the web page:
[image: image.png]

This button is the one that "clicked (flickered)" but nothing after that.
I just now tried the email page in both Chrome and Edge and this web page
"Welcome to Caddy Community" page in Chrome and Edge.  Neither browser
version worked.  I also just now tried the Welcome page in a Raspberry Pi
Chromium browser.  No luck there either.

VBR,

Dennis
